rmf_demos_tasks/rmf_demos_tasks/dispatch_cart_delivery.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+
+# Copyright 2023 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import uuid
+import argparse
+import json
+import asyncio
+
+import rclpy
+from rclpy.node import Node
+from rclpy.parameter import Parameter
+from rclpy.qos import qos_profile_system_default
+from rclpy.qos import QoSProfile
+from rclpy.qos import QoSHistoryPolicy as History
+from rclpy.qos import QoSDurabilityPolicy as Durability
+from rclpy.qos import QoSReliabilityPolicy as Reliability
+
+from rmf_task_msgs.msg import ApiRequest, ApiResponse
+
+
+###############################################################################
+
+class TaskRequester(Node):
+
+    def __init__(self, argv=sys.argv):
+        super().__init__('task_requester')
+        parser = argparse.ArgumentParser()
+        parser.add_argument('-g', '--go_to', required=True,
+                            type=str,
+                            help="Go to start waypoint")
+        parser.add_argument('-p', '--pickup', required=True,
+                            type=str,
+                            help="Pickup waypoint name")
```

Comment:
The task now requires both `-g` for the place to go to and `-p` for the waypoint pickup name, however the `pickup` argument is not used.
I guess we should either add it as a parameter to the perform action (that however would do nothing in the current sim implementation) or remove one of them altogether